api.ts

```diff
@@ -1250,6 +1252,8 @@ export interface DynamicOptions {
   getSchema?: MetadataFormulaDef;
   /** See {@link DynamicSyncTableOptions.entityName} */
   entityName?: string;
+  /** See {@link DynamicSyncTableOptions.disableNewColumnsOnSyncByDefault} */
+  disableNewColumnsOnSyncByDefault?: boolean;
```

Comment:
Nitpick: I know @patrick-codaio recommended the `ByDefault` suffix in another comment, but I don't think it's necessary now. `OnSync` clarifies that this behavior only kicks in during the sync, and even though a user can manually add new columns that's not happening during the sync, but afterwards. I'd vote for dropping `ByDefault` for brevity, but I don't want to bike shed this too much :-)

Reply:
either way works for me. let me update.

Reply:
I also don't want to bike shed, I just want to make sure we're talking about the same thing - I wasn't worried about the pack maker thinking manual column adding would be disabled, I was worried the pack maker might think this end user toggle would be disabled:
![Screen Shot 2022-04-08 at 11 03 45 AM](https://user-images.githubusercontent.com/84040450/162497261-54ee4a5e-a14b-4da0-8733-710e4bc9605a.png)

If you still want to keep the "ByDefault" off, no biggie.

Reply:
Ah, I didn't realize that this is additionally controllable by the user. So this sync table field only sets the default toggle position for newly created tables?

Reply:
yes. it only sets the default value.

Reply:
Ah, now I understand the logic. I'm fine with either the current name (shorter) or what you had previous (more accurate).